test/integration/authenticationAPITests.js

```diff
@@ -20,6 +20,75 @@ import { config } from '../../src/config'
 const { SERVER_PORTS } = constants
 
 describe('API Integration Tests', () => {
+  describe('Retrieve Enabled Authentication types', () => {
+    let authDetails = null
+    const authConfig = config.authentication
+
+    before(async () => {
+      await testUtils.setupTestUsers()
+      authDetails = testUtils.getAuthDetails()
+      const startPromise = promisify(server.start)
+      await startPromise({ apiPort: SERVER_PORTS.apiPort })
+      await testUtils.setImmediatePromise()
+      await AuditModel.deleteMany({})
+    })
+
+    afterEach(async () => {
+      await AuditModel.deleteMany({})
+    })
+
+    beforeEach(() => {
+      config.authentication = authConfig
+    })
+
+    after(async () => {
+      await Promise.all([
+        testUtils.cleanupTestUsers(),
+        promisify(server.stop)()
+      ])
+    })
+
+    it('should only allow an admin user to retrieve authentication types', async () => {
+      await request(constants.BASE_URL)
+        .get('/authentication/types')
+        .set('auth-username', testUtils.nonRootUser.email)
+        .set('auth-ts', authDetails.authTS)
+        .set('auth-salt', authDetails.authSalt)
+        .set('auth-token', authDetails.authToken)
+        .expect(403)
+    })
+
+    it('should return an error when the authentication object is invalid', async () => {
+      config.authentication = {}
+
+      await request(constants.BASE_URL)
+        .get('/authentication/types')
+        .set('auth-username', testUtils.rootUser.email)
+        .set('auth-ts', authDetails.authTS)
+        .set('auth-salt', authDetails.authSalt)
+        .set('auth-token', authDetails.authToken)
+        .expect(500)
+    })
+
+    it('should retrieve enabled authentication types', async () => {
+      config.authentication.enableMutualTLSAuthentication = true
+      config.authentication.enableBasicAuthentication = false
```

Comment:
@bradsawadye these current tests didn't pick up that the constant import declaration was incorrect. Can you also test the JWT and Basic auth